src/main/java/com/elasticbox/jenkins/util/ClientCache.java

```diff
@@ -71,6 +73,17 @@ public static final Client findOrCreateClient(String cloudName) throws IOExcepti
         return client;
     }
 
+    public static HttpProxy getJenkinsHttpProxyCfg() {
+        HttpProxy httpProxy = null;
+        ProxyConfiguration proxyConfiguration = Jenkins.get().proxy;
+        if ((proxyConfiguration != null) && (!StringUtils.isBlank(proxyConfiguration.name))) {
```

Comment:
Please use isNotBlank
```suggestion
        if ((proxyConfiguration != null) && (StringUtils.isNotBlank(proxyConfiguration.name))) {
```